client/FxAccountClient.js

```diff
@@ -791,14 +813,20 @@ define([
    * @param {String} email
    * @param {String} oldPassword
    * @param {String} newPassword
+   * @param {String} sessionToken
+   * @param {Object} [options={}] Options
+   *   @param {Boolean} [options.keys]
+   *   If `true`, calls the API with `?keys=true` to get the keyFetchToken
    * @return {Promise} A promise that will be fulfilled with JSON `xhr.responseText` of the request
    */
-  FxAccountClient.prototype.passwordChange = function(email, oldPassword, newPassword) {
+  FxAccountClient.prototype.passwordChange = function(email, oldPassword, newPassword, sessionToken, options) {
     var self = this;
+    options = options || {};
 
     required(email, 'email');
     required(oldPassword, 'old password');
     required(newPassword, 'new password');
+    required(sessionToken, 'sessionToken');
```

Comment:
@vbudhram - do you think I should make the sessionToken optional here? IIRC, our plan was to have the auth server accept the sessionToken as optional so that we can roll back the two servers independently. If it's optional, think it should go into the `options` block?